sdk/ai/azure-ai-projects/azure/ai/projects/_types.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    from . import models as _models
+    from .. import models as _models
```

Comment:
What is going on here?

Reply:
Thanks! I removed the 2nd line, which was added by Nikolay yesterday. Waiting to hear from him how the 2nd line got added...